src/database/migrations/20220327150708-create-like.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable require-jsdoc */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable('likes', {
```

Comment:
Add unique index to both user_id and facility_id, and it better to rename this fields to {userId, facilityId} to have a uniform in the database tables definitions